modules/tinymce/src/plugins/searchreplace/test/ts/module/test/Utils.ts

```diff
@@ -36,6 +39,15 @@ const pOpenDialog = async (editor: Editor) => {
   return await TinyUiActions.pWaitForDialog(editor);
 };
 
+const pOpenDialogWKeyboard = async (editor: Editor) => {
```

Comment:
```suggestion
const pOpenDialogWithKeyboard = async (editor: Editor) => {
```
This function name has a typo. It will need fixing where it's used as well.